Resources/views/CRUD/list_email.html.twig

```diff
@@ -0,0 +1,41 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+
+{% extends admin.getTemplate('base_list_field') %}
+
+{% block field%}
+    {%- if value is empty -%}
+        &nbsp;
+    {%- elseif field_description.options.as_string is defined and field_description.options.as_string -%}
+        {{ value }}
+    {%- else -%}
+        {%- set url = '' -%}
+        {%- if field_description.options.subject is defined and field_description.options.subject is not empty -%}
+            {%- set subject = field_description.options.subject -%}
+            {%- set url = 'subject=' ~ subject -%}
```

Comment:
> I read almost all templates for list variables like `list_url`, `list_date` and etc. Almost all templates have duplication code, so I try to keep save code style.

I understand. But this is not because we did some things in the past than we should not improve them. :wink: